web/components/FcDrawerRequestStudyBulkEdit.vue

```diff
@@ -102,17 +103,15 @@ export default {
   },
   methods: {
     actionSave() {
-      if (!this.formValid) {
-        const $form = this.$refs.formWrapper;
-        scrollToFirstError($form, '.v-input');
+      setTimeout(() => {
```

Comment:
wanna put some milliseconds on this? `setTimeout(fn, ms)`
is there a way to hook this up after the route change?

Reply:
If I put the timeout after the route change, then the emails field would start accepting bad values for emails. I'll give it some milliseconds though.

Reply:
is there a way to do `.then()` after the route change? I think evan has used `Vue.nextTick()` also.
I'm just worried that your fix will only work sometimes, depending on how fast the browser/network is being.

Reply:
Hmm, that is true. I'm currently looking to into using refs and nextTick. Haven't gotten it to work yet though.

Reply:
nice one!!